java/short_url_generator/README.md

```diff
@@ -0,0 +1,56 @@
+# 💻 Get Short URL
+
+A Java Cloud Function for generating a Short URL using [tinyurl](https://tinyurl.com/app) and [bitly](https://bitly.com/)
+
+_Example input:_
+
+```json
+{
+    "provider": "tinyurl",
+    "url": "https://google.com"
+}
+```
+
+Supported providers: tinyurl, bitly
+
+_Example output:_
```

Comment:
Let's please have one more output example where`success`  is `false`, and there is a `message`.